runtime/doc/builtin.txt

```diff
@@ -4261,6 +4262,23 @@ getreginfo([{regname}])					*getreginfo()*
 		Can also be used as a |method|: >
 			GetRegname()->getreginfo()
 
+getregion({pos1}, {pos2})				*getregion()*
+		Returns the list of strings from {pos1} to {pos2}.
+		For possible values of {pos1} and {pos2} see |line()|.
+		It is useful to get text starting and ending in different
+		columns, such as |characterwise-visual| selection.
+
+		Note that
+		- {pos1} and {pos2} are swapped if {pos1} is after {pos2}.
```

Comment:
thx! minor thing about typography (I don't know what vim help prefers), but either way list items should be separated by `,` or `;` and/or `.` for the last list item.

Reply:
I have checked the other documentation.  It seems terminated by "." at all.